agreement/index.md

```diff
@@ -53,27 +52,25 @@ Have agreed as follows:
 
 ## Article 1
 
-For the purposes of this Agreement, all definitions contained in Article 1 of the Convention apply. In addition:
+For the purpose of this Agreement, the definitions contained in Article 1 of the Convention shall apply. In addition:
 
-1. "Convention" means the United Nations Framework Convention on Climate Change, adopted in New York on 9
-May 1992.
-2. "Conference of the Parties" means the Conference of the Parties to the Convention.
-3. "Party" means a Party to this Agreement
+a. "Convention" means the United Nations Framework Convention on Climate Change, adopted in New York on 9 May 1992;
+b. "Conference of the Parties" means the Conference of the Parties to the Convention;
+c. "Party" means a Party to this Agreement.
 
 ## Article 2
 
 1. This Agreement, in enhancing the implementation of the Convention, including its objective, aims to strengthen the global response to the threat of climate change, in the context of sustainable development and efforts to eradicate poverty, including by:
 
-    1. Holding the increase in the global average temperature to well below 2 °C above pre-industrial levels and to pursue efforts to limit the temperature increase to 1.5 °C above pre-industrial levels, recognizing that this would significantly reduce the risks and impacts of climate change;
```

Comment:
Ok, I can change that to always use numbers for sub-lists as well. I was reluctant to use it right away because the editorial corrections have an example of changing numbers to letters.

But it's probably the easiest way to use numbers, so I will add this.